wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController+SearchResults.swift

```diff
@@ -131,7 +127,7 @@ extension StartUIViewController: SearchResultsViewControllerDelegate {
     }
 
     func createGuestRoom() {
-        guard let userSession = ZMUserSession.shared() else {
+        guard let userSession = userSession as? ZMUserSession else {
```

Comment:
Because `userSession` here is of type `any UserSession`, which doesn't expose the `viewContext` property needed below. That's why we need to cast to `ZMUserSession`

Reply:
Can we add a TODO comment so we can fix it on a later stage?

Reply:
what would you like to fix later?

Reply:
I think we should avoid to cast back to `ZMUserSession` by expanding the API of `UserSession`.

Reply:
> I think we should avoid to cast back to ZMUserSession by expanding the API of UserSession. @johnxnguyen 

@David-Henner That's what I wanted the todo for. ☝️ We can definitely do it on a separate PR.

Reply:
done https://github.com/wireapp/wire-ios/pull/729/commits/3c218a48044ccf7e46d88cc0db69242c3904a8f6